### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by IanCStewart**

### CODE REVIEW

File: assignment5/static/app.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*global window, document*/
+(function (){
+  const section = {
+    toggle: function (){
+      const route = window.location.hash;
+      if(route === '') {
```
Comment (TuriGuilano) [inline assignment5/static/app.js]: Call een init()!

**Review by TuriGuilano — CHANGES_REQUESTED** [review]
https://github.com/IanCStewart/minor-wafs/pull/2/files#diff-82c6855f39420678d876d3ceb01e5121R6
should be an init: function { ......

File: assignment5/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    </p>
+    <p>
+      Pig picanha turducken doner bresaola chuck shank. Turducken kevin burgdoggen alcatra landjaeger, meatball flank ribeye bacon pork loin kielbasa. Kevin bresaola andouille corned beef cupim. Kevin jowl salami, swine tail short ribs bacon.
+    </p>
+    <p>
+      Ground round jowl pancetta beef ribs. Rump chuck tail alcatra ham kevin turkey tenderloin porchetta pastrami pig cupim tongue beef ribs. Bresaola t-bone short ribs shankle ribeye ground round shoulder pork loin turkey. Ham hock beef ribs turducken, frankfurter cow picanha short ribs ball tip capicola turkey tenderloin cupim pork loin bacon. Frankfurter venison pork loin boudin jerky, short loin ground round kevin tri-tip ham hock. Meatball venison pig frankfurter strip steak tri-tip pork loin corned beef.
+    </p>
+  </section>
+</body>
+<script src="./static/app.js"></script>
```
Comment (rijkvanzanten) [inline assignment5/index.html]: Script tag outside of body [won't validate](https://validator.w3.org/nu/?doc=https%3A%2F%2Fiancstewart.github.io%2Fwafs%2F).

File: assignment5/static/app.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+/*global window, document*/
+(function (){
+  const section = {
+    init: function (){
```
Comment (rijkvanzanten) [inline assignment5/static/app.js]: You could use the ES6 methods

f.e.
```
const section = {
  init() {
    ...
  }
}
```

File: assignment5/static/app.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      routes.pages.forEach(function (page) {
+        `#${page}` === route
+        ? document.getElementById(page).classList.remove('invisible')
+        : document.getElementById(page).classList.add('invisible');
+      });
+    }
+  };
+  const routes = {
+    pages: [
+      'home',
```
Comment (rijkvanzanten) [inline assignment5/static/app.js:31]: You could've put the hashes in here already. Then you wouldn't have to this to add the hash later.
```
`#${page}` === route
```

File: assignment5/static/style.css [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+* {
+  font-family: 'Lato', sans-serif;
+  font-weight: normal;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  width: 100%;
```
Comment (rijkvanzanten) [inline assignment5/static/style.css]: width, height 100% doesn't do anything in this situation

File: assignment5/static/style.css [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  display: block;
+}
+
+nav a:hover {
+  background: #467BBE;
+  color: #efefef;
+}
+
+.invisible {
+  height: 0;
```
Comment (rijkvanzanten) [inline assignment5/static/style.css]: Why `height: 0;` when you remove the whole element with `display: none;`

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]
Please see my comments. Yours sincerely, je boy Rijk

**Pull request merged**